src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml

```diff
@@ -0,0 +1,26 @@
+@using OrchardCore.Forms.Models
+@model ShapeViewModel<SelectPart>
+@{
+    var formElementPart = Model.Value.ContentItem.As<FormElementPart>();
+    var formInputElementPart = Model.Value.ContentItem.As<FormInputElementPart>();
+    var elementId = formElementPart.Id;
+    var fieldName = formInputElementPart.Name;
+    var fieldId = !string.IsNullOrEmpty(elementId) ? elementId : !string.IsNullOrEmpty(fieldName) ? Html.GenerateIdFromName(fieldName) : default(string);
+    var fieldValue = Model.Value.DefaultValue;
+    var fieldClass = "form-control";
+
+    if (ViewData.ModelState.TryGetValue(fieldName, out var fieldEntry))
+    {
+        fieldValue = fieldEntry.AttemptedValue;
```

Comment:
It could be  - a joined into a comma-separated string for multiple select

Reply:
Not sure I understand what you mean here?

It supports only a single select, a multi select would be another field, with a different name

Reply:
Nervermind